src/dbwrapper.cpp

```diff
@@ -147,6 +147,7 @@ static leveldb::Options GetOptions(size_t nCacheSize)
         // on corruption in later versions.
         options.paranoid_checks = true;
     }
+    options.max_file_size = std::max(options.max_file_size, DBWRAPPER_MAX_FILE_SIZE);
```

Comment:
As mentioned in the comments, it seems to me that `16` may be a better default value based on the measured IBDs - basically just as fast as `128`, without having to worry about the increase in e.g.  `MaxGrandParentOverlapBytes` and `ExpandedCompactionByteSizeLimit` (10x and 25x this value) called e.g. in `IsTrivialMove` with a warning: `"the move could create a parent file that will require a very expensive merge later on"` (or any other such surprise) - which we likely want to avoid:
```suggestion
    options.max_file_size = 16 << 20;
```

Reply:
Thanks, please resolve the comment